probe-rs/src/bin/probe-rs/cmd/dap_server/debug_adapter/dap/repl_commands.rs

```diff
@@ -361,4 +362,55 @@ pub(crate) static REPL_COMMANDS: &[ReplCommand<ReplHandler>] = &[
             memory_read(input_address, gdb_nuf, target_core)
         },
     },
+    ReplCommand {
+        command: "dump",
+        help_text: "Create a core dump at a target location",
+        sub_commands: None,
+        args: Some(&[
+            ReplCommandArgs::Optional("memory start address"),
+            ReplCommandArgs::Optional("memory size in bytes"),
+        ]),
+        handler: |target_core, command_arguments, _request_arguments| {
+            let input_arguments = command_arguments.split_whitespace();
+            let location = Path::new("./coredump");
```

Comment:
Same comment as before ... it would be nice to not be bound by a hard coded value.